lib/customer.rb

```diff
@@ -0,0 +1,45 @@
+require "csv"
+require "ap"
+require "pry"
+
+class Customer
+  attr_reader :id, :email, :address
+  @@customers = []
```

Comment:
This syntax `@@customers` is a class variable -- this ends up being a _very_ practical and correct way to use a class variable in Ruby, but class variables are pretty powerful if not used correctly! Just a warning, although, if there's _any_ reason to use a class variable, this is the most correct and safe way to use it :)